tests/test_parser.py

```diff
@@ -509,3 +510,28 @@ def test_term_extraction(self):
         # simple query with pipes
         simple_extracted = extract_query_terms(network_event + "| unique process_name, user_name\n\n| tail 10")
         self.assertListEqual(simple_extracted, [network_event.strip()])
+
+    def test_elasticsearch_flag_enabled(self):
+        """Check that removed Endgame syntax throws an error and new syntax does not."""
+        schema = Schema({"process": {"process_name": "string",  "pid": "number"}})
+
+        with elasticsearch_syntax, schema:
+            parse_query('process where process_name : "cmd.exe"')
+            parse_query('process where process_name : """cmd.exe"""')
+            parse_query('process where process_name : """""cmd.exe"""""')
```

Comment:
want to add some method tests ` :length()` too?

Reply:
there are some other unit tests that check for methods elsewhere.
or are you saying that we should make sure that methods are _not_ allowed with `elasticsearch_syntax`?

Reply:
actually, maybe I am confused. From your description:

>So now, there is no whitespace allowed between : the name of the method and the open parenthesis. :length() is valid, but : length() is not valid and :length () is not either. But you can still have space before the colon for methods.

are methods allowed with `:` comparisons?

Reply:
python will still recognize methods, but it's now a little stricter when it looks for what looks like a method. here's the  current behavior

```
foo : "bar"     // insensitive version of `foo == "bar"`
foo : length   // insensitive version of `foo == length`
foo : length()  // method version of `length(foo)`
foo : length () // syntax error
```

Reply:
cool, so the original suggestion was just to ensure case 4 raised the error - your call

Reply:
can do.

Reply:
fyi, this is actually a _semantic_ error. glad we added tests.

`foo : length()` is the case insensitive version of `foo == length()`